lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -30,6 +30,10 @@
       - Use with state C(present) to archive an image to a .tar file.
     required: false
     version_added: "2.1"
+  cache_from:
```

Comment:
Should include `required`, `type`, and `default`

Reply:
I have update code with `required` ans `type` here. Is it require to mention `default: None` here?

Reply:
Nope. Only need to specify default if it's non-null.

Reply:
Always having `type` is a good idea; always having `required` is not really necessary from my point of view, and `default` should only be there if `required` is `False` and it is something "interesting" (i.e. non-null).